static/js/components/CommentTree.js

```diff
@@ -51,196 +53,254 @@ type Props = {
+              <li className="comment-action-button report-count">
+                Reports: {comment.num_reports}
+              </li>
             ) : null}
-            {atMaxDepth ||
-            moderationUI ||
-            comment.deleted ||
-            userIsAnonymous() ? null : (
+            {SETTINGS.username === comment.author_id && !moderationUI ? (
+              <li>
                 <div
-                  className="comment-action-button reply-button"
+                  className="comment-action-button edit-button"
                   onClick={e => {
                     if (beginEditing) {
-                      beginEditing(formKey, initialValue, e)
+                      beginEditing(editFormKey, comment, e)
                     }
                   }}
                 >
-                  <a href="#">reply</a>
+                  <a href="#">edit</a>
                 </div>
-              )}
-            {userIsAnonymous() ? null : this.renderFollowButton(comment)}
-            {comment.num_reports ? (
-              <div className="comment-action-button report-count">
-                Reports: {comment.num_reports}
-              </div>
-            ) : null}
-            {SETTINGS.username === comment.author_id && !moderationUI ? (
-              <div
-                className="comment-action-button edit-button"
-                onClick={e => {
-                  if (beginEditing) {
-                    beginEditing(editFormKey, comment, e)
-                  }
-                }}
-              >
-                <a href="#">edit</a>
-              </div>
+              </li>
             ) : null}
             {SETTINGS.username === comment.author_id && deleteComment ? (
-              <div
-                className="comment-action-button delete-button"
-                onClick={preventDefaultAndInvoke(() => deleteComment(comment))}
-              >
-                <a href="#">delete</a>
-              </div>
+              <li>
+                <div
+                  className="comment-action-button delete-button"
+                  onClick={preventDefaultAndInvoke(() =>
+                    deleteComment(comment)
+                  )}
+                >
+                  <a href="#">delete</a>
+                </div>
+              </li>
             ) : null}
             {comment.num_reports && ignoreCommentReports ? (
-              <div
-                className="comment-action-button ignore-button"
-                onClick={preventDefaultAndInvoke(() =>
-                  ignoreCommentReports(comment)
-                )}
-              >
-                <a href="#">ignore all reports</a>
-              </div>
+              <li>
+                <div
+                  className="comment-action-button ignore-button"
+                  onClick={preventDefaultAndInvoke(() =>
+                    ignoreCommentReports(comment)
+                  )}
+                >
+                  ignore all reports
```

Comment:
Should this still be within an `&lt;a href="#"&gt; tag?  Looks different without it (gray text, not aligned with other options in dropdown).

![screen shot 2018-08-24 at 10 10 51 am](https://user-images.githubusercontent.com/187676/44589251-0ff57180-a786-11e8-9832-ac5dbc5f16aa.png)

Reply:
ah yes it should! I had refactored to remove all the `<a />` tags but then realized that they looked weird, I must have missed putting this one back